pkgs/development/python-modules/justext/default.nix

```diff
@@ -0,0 +1,40 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, lxml
+# testing
+, pytest
+, coverage
+, pytest-cov
+}:
+
+buildPythonPackage rec {
+  pname = "justext";
+  version = "3.0.0";
+  format = "setuptools";
+
+  src = fetchPypi {
+    pname = "jusText";
+    inherit version;
+    sha256 = "sha256-dkDiSCGHlfa+ZfbDX+aXMloygPy0Z10VJbzf8rhvqt8=";
+  };
+
+  propagatedBuildInputs = [
+    lxml
+  ];
+
+  nativeCheckInputs = [
+    pytest
+    pytest-cov
+    coverage
```

Comment:
Coverage doesn't add value for us. Please patch it out. Check `charset-normalizer` for a hint.